sys-block/open-iscsi/files/iscsid.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Open-iSCSI
+Documentation=man:iscsid(8) man:iscsiuio(8) man:iscsiadm(8)
+After=network.target NetworkManager-wait-online.service iscsiuio.service tgtd.service targetcli.service
```

Comment:
We can, but we're divergent from upstream in that case - After= is only relevant if those other services exist and are enabled, it'll still start fine without them.